### PR TITLE
Feature/z candybar

### PR DESCRIPTION
**Pull request by ufollettu**
# Component - z-candybar - agnostic notification bar

----

### Motivation and Context

This is an agnostic component that can be used as a notification bottom bar. It can be used in association with custom content or with the z-binding-message component

### Priority

- [ ] 1 - Highest
- [ ] 2 - High
- [X] 3 - Medium
- [ ] 4 - Low
- [ ] 5 - Lowest
- [ ] 6 - Not a Priority

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [X] Component (add a Component as approved dy Design System)
- [ ] Docs (add documentation)
- [ ] Chore (changes that adds small enhancement)
- [ ] Breaking (fix or feature that would cause existing functionality to not work as expected)

### Abstract and Screenshots

Abstract: https://app.abstract.com/projects/fd370780-e659-11e8-99dc-0d08537c5fde/branches/c40f8f25-f562-4870-a0ec-b1461ac9bf53/commits/latest/files/9f586aa0-bd5b-4c12-9541-aed2e98a7a9c/layers/1EBAC532-79B0-4D4D-BAEA-04FEC74FC026?collectionId=4b9de467-8f94-414f-a16e-c0a62bc44f81&collectionLayerId=b0b887ad-f141-4d50-b49e-b0d6414666a8

### Note

----

### CODE REVIEW

File: src/components/notification/z-candybar/styles.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  background: var(--myz-blue-lighter);
+  border-radius: var(--half-x1);
+  box-shadow: 0px -2px 4px 0px rgba(66, 69, 72, 0.4);
+}
+
+/* Tablet breakpoint */
+@media only screen and (min-width: 768px) {
+  :host > div {
+    position: initial;
+    display: flex;
```
Comment (a-sartori-wt) [inline src/components/notification/z-candybar/styles.css:33]: flex-direction, flex-wrap sono ereditati dal padre

Comment (ufollettu) [reply src/components/notification/z-candybar/styles.css]: fixed

File: src/components/notification/z-candybar/styles.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  justify-content: center;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: auto;
+  height: auto;
+  max-height: 250px;
+  z-index: 1000;
+  margin: 28vh 8vw;
```
[the remark this answers is not in the document]
Comment (ufollettu) [reply src/components/notification/z-candybar/styles.css]: fixed (margin is auto now)

File: src/components/notification/z-candybar/styles.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: auto;
+  height: auto;
+  max-height: 250px;
```
[the remark this answers is not in the document]
Comment (ufollettu) [reply src/components/notification/z-candybar/styles.css]: fixed to 232px

File: src/components/notification/z-candybar/styles.css [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  z-index: 1000;
+  margin: 28vh 8vw;
+  overflow: auto;
+  overflow: -moz-scrollbars-none;
+  -ms-overflow-style: none;
+  background: var(--myz-blue-lighter);
+  border-radius: var(--half-x1);
+  box-shadow: 0px -2px 4px 0px rgba(66, 69, 72, 0.4);
```
[the remark this answers is not in the document]
Comment (ufollettu) [reply src/components/notification/z-candybar/styles.css:26]: si, ha senso, al momento non sono presenti in token del DS, li possiamo definire e implementare in una storia specifica

**Review by a-sartori-wt — APPROVED** [review]

**Review by walidl — APPROVED** [review]

**Pull request merged**